src/pages/Transfer/TransferOverview/index.tsx

```diff
@@ -0,0 +1,3 @@
+import TransferOverview from './TransferOverview';
```

Comment:
This naming doesn't feel right—overview  suggests we're showing data or a summary (like we do with the vaults), but we're not. I think it the structure we had before made sense, with an index page rendering the tabbed view, and sub-directories for the transfer/XCM transfer form, and a components directory for anything shared.

Reply:
But the structure before did not following what we have been doing because XCM does not have a page on his own. Both transfer and xcm exist on the same page. I tried naming it Transfer but since the parent is also Transfer, it made the file system buggy. I might change the name from TransferOverview to `TransferMethods` or `TransferForms`

Reply:
That's not quite what I was getting at—what I mean is we have this:

```
const Transfer = (): JSX.Element => {
  return <TransferOverview />;
};
```

And I'm not sure what advantage there is to making a `<TransferOverview>` component rather than having:

```
 const Transfer = (): JSX.Element => {
  return (
    <MainContainer>
      <StyledWrapper>
        <StyledCard gap='spacing2'>
          <Flex direction='column' gap='spacing8'>
            <Tabs size='large' fullWidth>
              <TabsItem title='Transfer' key='transfer'>
                <StyledFormWrapper>
                  <TransferForm />
                </StyledFormWrapper>
              </TabsItem>
              <TabsItem title='Cross Chain Transfer' key='crossChainTransfer'>
                <StyledFormWrapper>
                  <CrossChainTransferForm />
                </StyledFormWrapper>
              </TabsItem>
            </Tabs>
          </Flex>
        </StyledCard>
      </StyledWrapper>
    </MainContainer>
  );
};
```